src/driver/hyperv.rs

```diff
@@ -255,12 +255,6 @@ impl HyperV {
     }
 }
 
-impl Introspectable for HyperV {
```

Comment:
I think you need to leave an empty Impl for Hyper-V, where all methods returns unimplemented(), otherwise the CI fails

Reply:
Actually, this wasn't intentional. I did not want to delete the whole impl block. :scream: